packages/ui/src/Bento/elements/tables/Basic.tsx

```diff
@@ -58,14 +60,18 @@ export function BasicTable({
   refetch,
   setRefetch,
 }: BasicTableProps) {
+  const { user } = useProfile(null);
```

Comment:
we have useAuthUser hook for getting the user. please use it here

Reply:
Sure